### PR TITLE
Add 5.7 CI configurations and tests in OAL and ES-proxy 

### PR DESCRIPTION
**Pull request by btaani**
Update CI configurations and tests of origin-aggregated-logging and elasticsearch-proxy to remove 5.4 and add 5.7 base images

Ref: [LOG-4248](https://issues.redhat.com/browse/LOG-4248)

/cc @jcantrill @periklis

### CODE REVIEW

**Review by periklis — APPROVED** [review]

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml [hunk 1]
```diff
@@ -11,8 +7,8 @@ base_images:
     name: "5.6"
     namespace: logging
     tag: cluster-logging-operator-e2e
```
Comment (xperimental) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml:9]:
```suggestion
    tag: cluster-logging-operator
```

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml [hunk 2]
```diff
@@ -11,8 +7,8 @@ base_images:
...
-  cluster-logging-operator-e2e-5-4:
-    name: "5.4"
+  cluster-logging-operator-5-7:
+    name: "5.7"
     namespace: logging
     tag: cluster-logging-operator-e2e
```
Comment (xperimental) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml:13]:
```suggestion
    tag: cluster-logging-operator
```

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml [hunk 3]
```diff
@@ -224,22 +224,22 @@ tests:
...
     workflow: ipi-aws
-- as: smoke-5-6
+- as: smoke-5-7
   steps:
     cluster_profile: aws
     test:
     - as: test
       cli: latest
-      commands: LOGGING_VERSION=5.6 CLO_BRANCH=master EO_BRANCH=master make test
+      commands: LOGGING_VERSION=5.7 CLO_BRANCH=master EO_BRANCH=master make test
```
Comment (xperimental) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml:233]:
```suggestion
      commands: LOGGING_VERSION=5.7 CLO_BRANCH=release-5.7 EO_BRANCH=release-5.7 make test
```

**Comment by xperimental** [comment]
/pj-rehearse ci/rehearse/openshift/origin-aggregated-logging/master/cluster-logging-operator-e2e-5-5 ci/rehearse/openshift/origin-aggregated-logging/master/cluster-logging-operator-e2e-5-6

**Comment by xperimental** [comment]
/pj-rehearse pull-ci-openshift-origin-aggregated-logging-master-cluster-logging-operator-e2e-5-5 pull-ci-openshift-origin-aggregated-logging-master-cluster-logging-operator-e2e-5-6

**Comment by btaani** [comment]
/pj-rehearse pull-ci-openshift-origin-aggregated-logging-master-cluster-logging-operator-e2e-5-5

**Pull request merged**